src/app/legacy/psammead/psammead-social-embed/src/CaptionWrapper/index.jsx

```diff
@@ -34,40 +35,40 @@ const FigCaption = styled.figcaption`
 const CaptionWrapper = ({
   children,
   service,
-  textPrefixVisuallyHidden,
   text,
   additionalText,
+  descById,
 }) => {
   const { pageType } = useContext(RequestContext);
   const isLive = pageType === LIVE_PAGE;
   const isTransparentPage = pageType === ARTICLE_PAGE || isLive;
 
   return (
-    <Figure isTransparentPage={isTransparentPage}>
+    <Container isTransparentPage={isTransparentPage}>
       {children}
-      <FigCaption
+      <WarningText
+        {...(descById && { id: descById })}
         isTransparentPage={isTransparentPage}
         isLive={isLive}
         service={service}
       >
-        {textPrefixVisuallyHidden && <span>{textPrefixVisuallyHidden}</span>}
         {`${text}${additionalText ? ` ${additionalText}` : ''}`}
-      </FigCaption>
-    </Figure>
+      </WarningText>
+    </Container>
   );
 };
 
 CaptionWrapper.defaultProps = {
-  textPrefixVisuallyHidden: null,
   additionalText: null,
+  descById: null,
 };
 
 CaptionWrapper.propTypes = {
   children: node.isRequired,
   service: string.isRequired,
-  textPrefixVisuallyHidden: string,
   text: string.isRequired,
   additionalText: string,
+  descById: string,
```

Comment:
Could we use a more meaningful variable name here?

```suggestion
  describedById: string,
```